packages/framework/src/types/schema.types/json.schema.types.ts

```diff
@@ -6,7 +6,11 @@ import type { JSONSchema, FromSchema as JsonSchemaInfer } from 'json-schema-to-t
  * This type is used to narrow the type of a JSON schema to a minimal type
  * that is compatible with the `json-schema-to-ts` library.
  */
-export type JsonSchemaMinimal = { type: 'object' } | { anyOf: unknown[] } | { allOf: unknown[] } | { oneOf: unknown[] };
+export type JsonSchemaMinimal =
+  | { type: unknown }
```

Comment:
All base schema inference now allow for any schemas, disallowing primitives are handled in the base schema entry point to keep each schema library typing simple.